.github/workflows/verify.yml

```diff
@@ -40,8 +37,27 @@ jobs:
 
       - name: Check manifests are up-to-date
         run: |
-          ./manifests/hivemq-operator/manifests.sh
-          ./manifests/hivemq-platform-operator/manifests.sh
-          ./manifests/hivemq-platform/manifests.sh
-          ./manifests/hivemq-swarm/manifests.sh
+          git add manifests/ -v
```

Comment:
I think the `git add` command must be executed after the manifest scripts, otherwise newly created files will not be staged and the `git diff` will still not fail.

Reply:
good catch